### DISCUSSION TITLE
my github page was stolen by a porn site

### DISCUSSION THREAD

**Discussion by love1angel** [community/community#59070] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I used to bug domain service to direct my github page love1angel.github.io/ to pengxie.xyz, when my domain service expires, i found although i delete the repo, but when i typed love1angel.github.io/ in browser, it can still redirect to pengxie.xyz which now is a porn site, how can i prevent from this

**Comment by airtower-luna** [comment, marked as answer]
You need to remove the custom domain from the repository, the easiest way should be to delete the [`CNAME` file](https://github.com/love1angel/love1angel.github.io/blob/master/CNAME).

**Reply by love1angel** [reply]
thanks, got it

**Reply by airtower-luna** [reply]
You didn't need to delete the whole repository, just the file. :sweat_smile:
